test/spec/adapters/adsupply_spec.js

```diff
@@ -0,0 +1,359 @@
+describe('adsupply adapter tests', function () {
+
+	const expect = require('chai').expect;
+	const assert = require('chai').assert;
```

Comment:
const assert = require('chai').assert;    - REMOVED


const expect = require('chai').expect; - USED in the first test